packages/flutter/test/material/slider_test.dart

```diff
@@ -3703,4 +3703,209 @@ void main() {
       );
     });
   });
+
+  group('Slider.allowedInteraction', () {
+    testWidgets('is tapOnly', (WidgetTester tester) async {
```

Comment:
```suggestion
    testWidgets('SliderInteraction.tapOnly', (WidgetTester tester) async {
```
Can you title these based on value `allowedInteraction` enum value